src/components/Post/Comment/CommentPanel.tsx

```diff
@@ -35,7 +69,10 @@ const CommentPanel = ({ post }: { post: Post | undefined }) => {
                                 </Box>
                             );
                         })
+                        
                     }
+                    {hasMoreComments && 
+                        <Button onPress={fetchMoreComments}><Text variant='button'>Load More</Text></Button>}
```

Comment:
lmao, this is good. make them load them if they need em with a button. nice.

Reply:
Figured I'd start with a simple solution before trying to create the scroll hook